app/src/main/java/com/example/kursakademiaandroida/features/data/remote/model/CharacterRemote.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.kursakademiaandroida.features.data.remote.model
+
+import com.example.kursakademiaandroida.features.characters.domain.model.Character
+import com.example.kursakademiaandroida.features.characters.domain.model.CharacterLocation
+import com.example.kursakademiaandroida.features.characters.domain.model.Origin
+import com.google.gson.annotations.SerializedName
+
+
+data class CharacterRemote(
+    @SerializedName("id") val id: Int,
+    @SerializedName("name") val name: String,
+    @SerializedName("status") val status: String,
+    @SerializedName("species") val species: String,
+    @SerializedName("type") val type: String,
+    @SerializedName("gender") val gender: String,
+    @SerializedName("origin") val origin: Origin,
```

Comment:
Dobrze by było, żeby klasy `Origin` oraz `CharacterLocation` miały swoje odpowiedniki na warstwie danych, analogicznie do tego jak to zostało zrobione z głównym obiektem `Character`.

Reply:
Z tego co widzę chyba nawet te klasy zostały już stworzone, wystarczy je tutaj użyć :)